.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: C
+
+install:
+  - CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - source eups-setups.sh
+  - setup lsst_sims
+  - pip install -r setup/requirements.txt
```

Comment:
If you implement the suggestion above, you can remove this line (and `requirements.txt`).